### PR TITLE
Allow to specify sentry envionment and release.

### PR DESCRIPTION
**Pull request by imperiuse**
Hi all!
PR for this issue -> https://github.com/aptible/supercronic/issues/114

### CODE REVIEW

**Comment by imperiuse** [comment]
From what I can see, the tests failed due to a dependency in the Go standard library. They can be fixed by upgrading the Go version. However, I'm not sure if such changes should be made by me or implemented in the current PR.

Can someone clarify what guidelines exist regarding this?

File: main.go
Comment (aaw) [inline main.go:43]: Remove the period at the end of the description for consistency with all the other flag descriptions

**Review by aaw — COMMENTED** [review]
Hey @imperiuse, thanks for the PR! Two things before merging:

1. Can you merge in [recent changes to master](https://github.com/aptible/supercronic/commit/65d24fd8fdb9da4336bdb371248a334f436fcba5)? I fixed the govulnreport build failure you've already noticed by upgrading the go version, that should make your build green.
2. Can you update the README? There's a [section describing the Sentry config](https://github.com/aptible/supercronic/blob/master/README.md?plain=1#L255) already that should be updated with this PR.

Thanks!

**Comment by imperiuse** [comment]
> Hey @imperiuse, thanks for the PR! Two things before merging:
> 
> 1. Can you merge in [recent changes to master](https://github.com/aptible/supercronic/commit/65d24fd8fdb9da4336bdb371248a334f436fcba5)? I fixed the govulnreport build failure you've already noticed by upgrading the go version, that should make your build green.
> 2. Can you update the README? There's a [section describing the Sentry config](https://github.com/aptible/supercronic/blob/master/README.md?plain=1#L255) already that should be updated with this PR.
> 
> Thanks!

done.

**Review by aaw — APPROVED** [review]
LGTM, thanks again!

**Comment by imperiuse** [comment]
> LGTM, thanks again!

Thank you for the review. Should I do anything else, or should I wait for the project maintainers?
P.S. This PR is important to me, I would like to get it in the release as soon as possible. Thx!

**Pull request merged**
